### DISCUSSION TITLE
Can't import useWalletClient from wagmi.

### DISCUSSION THREAD

**Discussion by roudra323** [wevm/wagmi#2294] [Q&A] [answered]
I am making a dapp boilerplate where I want the contract instance. So according to the Docs I wanted to import the "useWalletClient" from wagmi but can't find it. Other hooks like "useAccount" and "useBalance" works fine.
wagmi version ^0.12.13

**Comment by SaadNiaziDev** [comment, marked as answer]
useWalletClient is part of wagmi@1 i guess u have to work with useSigner if u haven't upgraded to wagmi@1

**Reply by roudra323** [reply]
Yes I found the solution. Thanks
